npm/src/db/db.ts

```diff
@@ -150,6 +134,9 @@ class DB implements DatabaseDriver {
 
   async close(): Promise<void> {
     await this.db.close();
+    clearInterval(this.statsIntervalId)
+    // Flush stats one last time
+    this.publishStats();
```

Comment:
Thanks @Reductions, this looks good. One comment though, shouldn't the last publishStats be done before closing the db?

Reply:
Done 👍🏻